net/ola/Makefile

```diff
@@ -75,7 +77,7 @@ endef
 
 define Host/Install
 	$(INSTALL_DIR) $(STAGING_DIR_HOSTPKG)/bin
-	$(CP) $(HOST_BUILD_DIR)/protoc/ola_protoc_plugin $(STAGING_DIR_HOSTPKG)/bin
+	$(INSTALL_BIN) $(HOST_BUILD_DIR)/protoc/ola_protoc_plugin $(STAGING_DIR_HOSTPKG)/bin
```

Comment:
This is not really related, isn't it? A separate commit would've been better, but ok with me.

Reply:
It is not related to macos build issue. If it is required I can split this commit to two commits